server/tests/flight.test.js

```diff
@@ -1,13 +1,106 @@
+import mongoMock from 'mongo-mock';
 import request from 'supertest-as-promised';
 import httpStatus from 'http-status';
 import chai from 'chai';
 import { expect } from 'chai';
 import app from '../../index';
+import should from 'should';
+let MongoClient = mongoMock.MongoClient;
+MongoClient.persist = 'mongo.js';
 
-chai.config.includeStack = true;
+describe('Flights', () => {
+  let connected_db;
+  let collection;
 
-describe('#Flight ', () => {
-  describe('# GET /flight/', () => {
+  before((done)=> {
+    MongoClient.connect("mongodb://localhost:27017/gomake ", (err, db)=> {
+      connected_db = db;
+      collection = connected_db.collection("flights");
+      done();
+    });
+  });
+
+  describe('indexes', function() {
+    it('should create a unique index on callSign and flightNumber',(done)=> {
+      collection.createIndex({
+        callSign: 1,
+        flightNumber: 1
+      }, {
+        unique: true
+      }, (err, name)=> {
+        if (err) {
+          return done(err)
+        };
+        name.should.equal('callSign_1_flightNumber_1');
+        done();
+      });
+    });
+
+    it('should deny unique constraint violations on insert', function(done) {
```

Comment:
Nice idea , if both of us are thinking on same lines then maybe a folder insides tests named flights where indexes.js would have the assertions for indexes etc etc .Just checking if you meant something like this or did you mean something else.